### PR TITLE
Peacemaker + gunslinger profession

### PR DESCRIPTION
**Pull request by SirPiecemaker**
#### Summary

```SUMMARY: Content "Mainline Colt Cattleman + Gunslinger profession"```

#### Purpose of change

I first made a gunslinger profession with the LeMat revolver, which didn't make that much sense for a performer. I then went and ported the Colt SAA from realguns in as a separate PR, but since I buggered that one up, I decided it be best if I close both and reopen them as a single PR - that way I can also make the Cattleman a part of the gunslinger's kit straight away. While the original intention was to add the Peacemaker, Probably not Tonkatsu reasonably proposed a modern reproduction of it - the Ubenti Cattleman - be more likely to be owned and found.

#### Describe the solution

Added a modern replica of the legendary SAA and a profession that starts with it.

#### Describe alternatives you've considered

The alternative was to make it an original antique, but the price of those is exceedingly rare and would be unlikely to be shot.

#### Additional context

Thanks to Probably Not Tonkatsu for lots of help with the matter.

### CODE REVIEW

File: data/json/items/gun/45colt.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+[
+  {
+    "id": "colt_saa",
+    "copy-from": "pistol_revolver",
+    "type": "GUN",
+    "name": "Uberti Cattleman",
+    "name_plural": "Uberti Cattleman",
+    "description": "This 7.5\" barreled Uberti Cattleman is a modern reproduction of the legendary Colt Single Action Army, or Colt Peacemmaker, one of the first revolvers to use a modern self-contained cartridge.  Made famous by westerns, the \"Peacemaker\" is still in demand for Cowboy Action Shooting, reenactors and collectors.  Unlike modern revolvers, the cylinder cannot swing out for loading, and spent brass must be ejected one at a time.",
```
Comment (matskuman5) [inline data/json/items/gun/45colt.json]:
```suggestion
    "description": "This 7.5\" barreled Uberti Cattleman is a modern reproduction of the legendary Colt Single Action Army, or Colt Peacemaker, one of the first revolvers to use a modern self-contained cartridge.  Made famous by westerns, the \"Peacemaker\" is still in demand for Cowboy Action Shooting, reenactors and collectors.  Unlike modern revolvers, the cylinder cannot swing out for loading, and spent brass must be ejected one at a time.",
```

File: data/json/itemgroups/ammo.json [hunk 2]
```diff
@@ -53,6 +52,7 @@
     "//": "Imported or otherwise very obscure pistol ammo.",
     "items": [
       [ "454_Casull", 100 ],
+      [ "45colt", 50 ],
```
Comment (mlangsdorf) [inline data/json/itemgroups/ammo.json]: 45colt is the amoo_type, not the item, which is 45colt_jhp or reloaded_45_colt_jhp.

File: data/json/professions.json [hunk 3]
```diff
@@ -3938,5 +3938,40 @@
...
+          "chaps_leather",
+          "gloves_leather",
+          "boots_western",
+          "duster",
+          "badge_deputy",
+          "sheriffshirt"
+        ],
+        "entries": [
+          { "item": "ear_plugs", "custom-flags": [ "no_auto_equip" ] },
+          { "item": "45colt", "charges": 18, "container-item": "bandolier_pistol" },
```
Comment (mlangsdorf) [inline data/json/professions.json]: again, 45colt_jhp not 45colt.

**Comment by tenmillimaster** [comment]
Just noticed, no pants on the profession either.  Jeans? Or breeches.

**Comment by SirPiecemaker** [comment]
> Just noticed, no pants on the profession either. Jeans? Or breeches.
> 
> Also still recommending the double barrel shotgun, and a pistol cartridge belt.

Pants were added. As for a shotgun, I think the revolver is quite alright for now - I can revisit it later. He does have a bandolier for the rounds.

File: data/json/items/gun/45colt.json [hunk 4]
```diff
@@ -0,0 +1,21 @@
+[
+  {
+    "id": "colt_saa",
+    "copy-from": "pistol_revolver",
+    "type": "GUN",
+    "name": "Uberti Cattleman",
+    "name_plural": "Uberti Cattleman",
+    "description": "This 7.5\" barreled Uberti Cattleman is a modern reproduction of the legendary Colt Single Action Army, or Colt Peacemaker, one of the first revolvers to use a modern self-contained cartridge.  Made famous by westerns, the \"Peacemaker\" is still in demand for Cowboy Action Shooting, reenactors and collectors.  Unlike modern revolvers, the cylinder cannot swing out for loading, and spent brass must be ejected one at a time.",
```
[the remark this answers is not in the document]
Comment (SirPiecemaker) [reply data/json/items/gun/45colt.json]: Changed.

[the remark this answers is not in the document]
Comment (tenmillimaster) [reply data/json/items/gun/45colt.json]: Sounds fine. Sorry for being nitpicky.

**Pull request merged**